### PR TITLE
Snake case 2

### PR DESCRIPTION
**Pull request by PatrikLundgren**
fixed comments on last pull request. Snake case in filehandler

### CODE REVIEW

File: ViAn/Filehandler/filehandler.cpp [hunk 1]
```diff
@@ -168,11 +168,11 @@ Project* FileHandler::load_project(std::string projname, std::string dirpath){
  * @param projFileStream
  * @return ID
  */
-ID FileHandler::load_project_file(std::string filePath, std::stringstream& projFileStream){
+ID FileHandler::load_project_file(std::string filepath, std::stringstream& proj_file_stream){
     std::string buf;
-    ID projFileID = add_file(filePath);
+    ID projFileID = add_file(filepath);
     read_file(projFileID, buf);
```
Comment (parso619) [inline ViAn/Filehandler/filehandler.cpp]: proj_file_ID

File: ViAn/Filehandler/filehandler.cpp [hunk 2]
```diff
@@ -4,9 +4,9 @@
  * @brief FileHandler::FileHandler
  */
 FileHandler::FileHandler() {
-    this->m_pid = 0; // zero out counter ids
-    this->m_fid = 0;
-    this->m_did = 0;
+    this->pid = 0; // zero out counter ids
+    this->fid = 0;
+    this->did = 0;
     this->lastError = 0;
```
Comment (GeorgeYildiz) [inline ViAn/Filehandler/filehandler.cpp]: lastError should be last_error

File: ViAn/Filehandler/filehandler.cpp (hunk 1 above)
Comment (GeorgeYildiz) [inline ViAn/Filehandler/filehandler.cpp]: agree

File: ViAn/Filehandler/filehandler.cpp [hunk 3]
```diff
@@ -214,11 +214,11 @@ void FileHandler::add_video(Project* proj, std::string filePath){
...
   */
 
-ID FileHandler::create_file(std::string filename, ID dirID){
+ID FileHandler::create_file(std::string filename, ID dir_id){
     std::ofstream f;
-    std::string filePath = this->get_dir(dirID)+"/"+filename;
-    f.open(filePath.c_str());
-    return this->add_file(filePath);
+    std::string filepath = this->get_dir(dir_id)+"/"+filename;
+    f.open(filepath.c_str());
```
Comment (GeorgeYildiz) [inline ViAn/Filehandler/filehandler.cpp]: should be file_path

File: ViAn/Filehandler/project.h [hunk 4]
```diff
@@ -15,7 +15,7 @@ typedef int ID;
  * Importand for seperation of different types of data.
  */
 struct ProjectStream{
-    std::stringstream projFile;
+    std::stringstream projfile;
```
Comment (GeorgeYildiz) [inline ViAn/Filehandler/project.h]: proj_file

File: ViAn/Filehandler/filehandler.cpp (hunk 2 above)
Comment (henpe071) [inline ViAn/Filehandler/filehandler.cpp]: p_id, f_id, d_id

File: ViAn/Filehandler/filehandler.cpp (hunk 3 above)
Comment (henpe071) [inline ViAn/Filehandler/filehandler.cpp]: should be file_name

File: ViAn/Filehandler/project.h [hunk 5]
```diff
@@ -15,7 +15,7 @@ typedef int ID;
...
     std::stringstream videos;
     std::stringstream analyzes;
```
Comment (henpe071) [inline ViAn/Filehandler/project.h]: analys

File: ViAn/GUI/mainwindow.cpp [hunk 6]
```diff
@@ -592,11 +592,11 @@ void MainWindow::on_actionLoad_triggered() {
...
-    projectInTree->setText(0, QString::fromStdString(proj->m_name));
+    MyQTreeWidgetItem *projectInTree = new MyQTreeWidgetItem(TYPE::PROJECT, QString::fromStdString(proj->name), proj->id);
+    projectInTree->setText(0, QString::fromStdString(proj->name));
     set_selected_project(projectInTree);
     ui->ProjectTree->addTopLevelItem(projectInTree);
-    for(Video *v: proj->m_videos) {
+    for(Video *v: proj->videos) {
         std::stringstream filePath;
         filePath << *v;
         std::string treeName = filePath.str();
```
Comment (henpe071) [inline ViAn/GUI/mainwindow.cpp]: fix snake_case

File: ViAn/Filehandler/filehandler.cpp [hunk 7]
```diff
@@ -98,7 +98,7 @@ void FileHandler::save_project(Project* proj){
 void FileHandler::update_proj_files(Project* proj){
     ProjectStream ps;
     ps << *proj;
-    write_file(proj->files->f_proj, ps.projFile.str(), WRITE_OPTION::OVERWRITE);
+    write_file(proj->files->f_proj, ps.projfile.str(), WRITE_OPTION::OVERWRITE);
```
Comment (fredrikiselius) [inline ViAn/Filehandler/filehandler.cpp]: proj_file

File: ViAn/Filehandler/filehandler.cpp [hunk 8]
```diff
@@ -119,9 +119,9 @@ void FileHandler::load_proj_files(std::string str){
  * @param dirpath
  * @return
  */
-Project* FileHandler::load_project(std::string fullProjectPath){
-    std::string dirpath = fullProjectPath.substr(0, fullProjectPath.find_last_of("/"));
-    std::string proj_name = fullProjectPath.substr(fullProjectPath.find_last_of("/")+1, fullProjectPath.length());
+Project* FileHandler::load_project(std::string full_project_path){
+    std::string dirpath = full_project_path.substr(0, full_project_path.find_last_of("/"));
```
Comment (fredrikiselius) [inline ViAn/Filehandler/filehandler.cpp]: dir_path

File: ViAn/Filehandler/filehandler.cpp [hunk 9]
```diff
@@ -136,28 +136,28 @@ Project* FileHandler::load_project(std::string fullProjectPath){
...
-    ProjectStream projStream;
+    add_project(std::make_pair(this->pid++, proj));
+    ProjectStream proj_stream;
 
     proj->saved = true;
 //    Read project file
-    std::string projFilePath = dirpath + "/" + projname + ".txt";
-    proj->files->f_proj = load_project_file(projFilePath, projStream.projFile);
+    std::string proj_filepath = dirpath + "/" + projname + ".txt";
+    proj->files->f_proj = load_project_file(proj_filepath, proj_stream.projfile);
```
Comment (fredrikiselius) [inline ViAn/Filehandler/filehandler.cpp]: file_path

File: ViAn/Filehandler/filehandler.cpp [hunk 10]
```diff
@@ -136,28 +136,28 @@ Project* FileHandler::load_project(std::string fullProjectPath){
...
 //    Read Analyzes
-    std::string analysesFilePath = dirpath + "/" + projname + "_analyses.txt";
-    proj->files->f_analysis = load_project_file(analysesFilePath, projStream.videos);
+    std::string analysis_filepath = dirpath + "/" + projname + "_analyses.txt";
+    proj->files->f_analysis = load_project_file(analysis_filepath, proj_stream.videos);
 //    Read Drawings
-    std::string drawingsFilePath = dirpath + "/" + projname + "_drawings.txt";
-    proj->files->f_drawings = load_project_file(drawingsFilePath, projStream.drawings);
+    std::string drawing_filepath = dirpath + "/" + projname + "_drawings.txt";
+    proj->files->f_drawings = load_project_file(drawing_filepath, proj_stream.drawings);
```
Comment (fredrikiselius) [inline ViAn/Filehandler/filehandler.cpp]: drawing_file_path

**Pull request merged**